app/views/psique/documentation/elements.html.haml

```diff
@@ -0,0 +1,218 @@
+  .col-3
+    .color-shape.dark-orange  
+      %span $dark-orange
+    %p.col-box.dark-grey.codehilite
+      For backgrounds you can use 
+      %span.nc .dark-orange
+      class 
+
+.main.grid
+  %p Secondary Colors
+  .col-3
+    .color-shape.grana
+      %span $grana
+    %p.col-box.dark-grey.codehilite
+      For backgrounds you can use 
+      %span.nc .grana
+      class 
+  .col-3
+    .color-shape.oceanic
+      %span $oceanic
+    %p.col-box.dark-grey.codehilite
+      For backgrounds you can use 
+      %span.nc .oceanic
+      class 
+
+.main.grid
+  .col-3
+    .color-shape.dark-magenta
+      %span $dark-magenta 
+    %p.col-box.dark-grey.codehilite
+      For backgrounds you can use 
+      %span.nc .dark-magenta
+      class 
+  .col-3
+    .color-shape.wisteria  
+      %span $wisteria
+    %p.col-box.dark-grey.codehilite
+      For backgrounds you can use 
+      %span.nc .wisteria
+      class 
+
+.grid.main
+  %h2.title Buttons
+  %p.main At the bottom of these lines, you can see the default colors that we use on our application.
+  
+  %p Buttons Shapes
+  .col-3
+    .btn--a Default Blue Button
+    %p.col-box.dark-grey.codehilite 
+      Add to your div
+      %span.nc .btn--a
+      class 
+  .col-3
+    .btn--b Default Orange Button
+    %p.col-box.dark-grey.codehilite 
+      Add to your div
+      %span.nc .btn--b
+      class 
+  .col-3
+    .btn--c Default Inactive Button
+    %p.col-box.dark-grey.codehilite 
+      Add to your div
+      %span.nc .btn--c
+      class 
+  .col-3
+    .btn--a.btn--icon
+      %i.fa.fa-github
+      Default Icon Button
+    %p.col-box.dark-grey.codehilite 
+      Add to your div
+      %span.nc .btn--icon
+      class 
+
+.grid.main
+  .col-4
+    .btn--a.btn--icon.btn--gplus
+      %i.fa.fa-google-plus
+      Compartir con Google +
+    %p.col-box.dark-grey.codehilite 
+      Add to your div
+      %span.nc .btn--gplus
+      class   
+  .col-4
+    .btn--a.btn--icon.btn--twitter
+      %i.fa.fa-twitter
+      Compartir con Twitter
+    %p.col-box.dark-grey.codehilite 
+      Add to your div
+      %span.nc .btn--twitter
+      class   
+  .col-4
+    .btn--a.btn--icon.btn--facebook
+      %i.fa.fa-facebook
+      Compartir con Facebook
+    %p.col-box.dark-grey.codehilite 
+      Add to your div
+      %span.nc .btn--facebook
+      class   
+.grid.main
+
+.grid.main
+  %p Button Sizes
+  .col-3
+    .btn--a.btn--l Large Button
+    %p.col-box.dark-grey.codehilite 
+      When your div has the class 
+      %span.nc .btn--a/b/c
+      append to this
+      %span.sc .btn--l  
+  .col-3
+    .btn--a.btn--s Small Button
+    %p.col-box.dark-grey.codehilite 
+      When your div has the class 
+      %span.nc .btn--a/b/c 
+      append to this
+      %span.sc .btn--s  
+  .col-3
+    .btn--a.btn--xs Extra Small Button
+    %p.col-box.dark-grey.codehilite 
+      When your div has the class 
+      %span.nc .btn--a/b/c
+      append to this
+      %span.sc .btn--xs
+
+.grid.main
```

Comment:
Don't track work in progress. A PR should show only finished work. :stuck_out_tongue: